pinot-core/src/main/java/com/linkedin/pinot/core/io/compression/PassThroughDecompressor.java

```diff
@@ -26,9 +25,11 @@
  */
 public class PassThroughDecompressor implements ChunkDecompressor {
   @Override
-  public int decompress(ByteBuffer inCompressed, ByteBuffer outDecompressed)
-      throws IOException {
+  public int decompress(ByteBuffer inCompressed, ByteBuffer outDecompressed) {
```

Comment:
rename variables -> compressedInput and decompressedOutput.

is this a bug fix?

Reply:
Yes, this is a bug fix. The bug would have only impacted no-dictionary column with variable width and pass-through compression (we don't have any live use cases for that).